overlord/snapstate/snapmgr.go

```diff
@@ -389,8 +390,15 @@ func refreshScheduleNoWeekdays(rs []*timeutil.Schedule) error {
 }
 
 func (m *SnapManager) checkRefreshSchedule() ([]*timeutil.Schedule, error) {
-	refreshScheduleStr := defaultRefreshSchedule
+	if m.refreshScheduleManaged() {
+		if m.currentRefreshSchedule != "managed" {
```

Comment:
Curious if this should exclude ability to manually manage a system. What do you think?

Reply:
Interessting idea. But also a wider scope than this PR :)